search-app/docs/api.md

```diff
@@ -3086,19 +3081,17 @@ The following parameters are supported when searching for variables.
 
 ##### Variable Matching Parameters
 
-These parameters will match fields within a variable. They are case insensitive by default. They support options specified. They also support searching with multiple values in the style of `variable_name[]=key1&variable_name[]=key2`. The values are ORed together.
+These parameters will match fields within a variable. They are case insensitive by default. They support options specified. They also support searching with multiple values in the style of `name[]=key1&name[]=key2`. The values are ORed together.
 
-* variable_name
-  * options: pattern, ignore_case
-* measurement
+* name
   * options: pattern, ignore_case
 * concept_id
 * keyword (free text)
   * keyword search is case insensitive and supports wild cards ? and *. There is a limit of 30 wild cards allowed in keyword searches. Within 30 wild cards, there's also limit on the max keyword string length. The longer the max keyword string length, the less number of keywords with wild cards allowed. The following fields are indexed for variable keyword search: variable name, measurement, and science keywords.
```

Comment:
ok so maybe the end of this line should be edited to replace measurement with long name